cmd/create.go

```diff
@@ -29,8 +30,19 @@ var componentCreateCmd = &cobra.Command{
 
 If component name is not provided, component type value will be used for the name.
 
+The component type will be chosen by default from current project,
+but if specific version is requested for, it'll be used from either
+* current
+   or
+* openshift
+
+project(with search precendence in same order) where available.
+So, if you would like it to be specifically from openshift project, please fully qualify the
+component type as $project/$component_type:$version.
+If version is not specified by default, latest wil be chosen as the version.
```

Comment:
This is way too long and you need to shorten it to one or two sentences.

Reply:
Ack